ppl/zqd/handlers_test.go

```diff
@@ -758,7 +761,7 @@ func TestIndexSearch(t *testing.T) {
 		},
 	})
 	require.NoError(t, err)
-	_, err = conn.LogPost(context.Background(), sp.ID, nil, babble)
+	_, err = conn.LogPost(context.Background(), sp.ID, nil, babbleSorted)
```

Comment:
Why change this?

Reply:
Because babble isn't sorted and this was causing archive to create overlaps, which are now getting compacted, and when chunks are compacted their indexes so their indices are getting wiped out. We don't plan on doing rebuilds of indices just yet.

Reply:
Instead of adding babbleSorted, how about doing something like this?
```go
	zcts := resolver.NewContext()
	zf, err := detector.OpenFile(zctx, babble, zio.ReaderOpts{})
	require.NoError(t, err)
	defer zf.Close()
	zrc, err := driver.NewReader(context.Background(), "sort ts", zctx)
	require.NoError(t, err)
	defer zrc.Close()
	_, err = conn.LogPost(context.Background(), sp.ID, nil, zrc)
```

Reply:
I've got to admit I find this unsavory.

Reply:
Alright I'll do it, this can go away anyways once #1656 is done.

Reply:
No offense taken if you decide it's too unsavory to swallow!